### ISSUE TITLE
TypeError: _filterizr2.default is not a function

### ISSUE THREAD

**Issue by Aidoboy** [giotiskl/filterizr#107]
All I really did was `npm install --save filterizr` and copy the getting started stuff. Any help?

This is the relevant Javascript from `app.js`:
```javascript
import $ from 'filterizr'

var filterizd = $('.filtr-container').filterizr({
    //options object
});
```

And here's `index.html`:
```html
<html>
<head>
    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
    <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
    <link rel="stylesheet" href="bundle.css">
    <script src="bundle.js" async></script>
    <meta http-equiv="X-Clacks-Overhead" content="GNU Terry Pratchett" />
</head>
<body>
<ul>
       <!-- For filtering controls add -->
       <li data-filter="all"> All items </li>
       <li data-filter="1"> Category 1 </li>
       <li data-filter="2"> Category 2 </li>
       <li data-filter="3"> Category 3 </li>
       <!-- For a shuffle control add -->
       <li data-shuffle> Shuffle items </li>
       <!-- For sorting controls add -->
       <li data-sortAsc> Ascending </li>
       <li data-sortDesc> Descending </li>
</ul>
<!-- To choose the value by which you want to sort add -->
<select data-sortOrder>
       <option value="domIndex"> Position </option>
       <option value="sortData"> Custom Data </option>
</select>
<div class="filtr-container">
        <div class="filtr-item" data-category="1" data-sort="value">
           <img src="img/sample.jpg" alt="sample">
        </div>
        <div class="filtr-item" data-category="2, 1" data-sort="value">
           <img src="img/sample.jpg" alt="sample">
        </div>
        <div class="filtr-item" data-category="1, 3" data-sort="value">
           <img src="img/sample.jpg" alt="sample">
        </div>
</div>
</body>
```

**Comment by wihodges** [comment]
Well this seems to work for me:  
window.$.filterizr = require('filterizr/dist/jquery.filterizr.min.js');

**Comment by Aidoboy** [comment]
I'm sorry, I don't remember.

**Comment by giotiskl** [comment]
Fixed in version 2.0.0 and above, bare in mind Filterizr is now a vanilla JS lib that can be used as a jQuery plugin, consultant the docs.
